.github/workflows/nix-linux.yml

```diff
@@ -42,5 +43,9 @@ jobs:
         with:
           name: hasktorch
           signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
-      - run: cachix use iohk
-      - run: nix-build
+      - run: |
+          cachix use iohk
+          nix-build -j 2
+          nix-build -j 2 -A haskellPackages.libtorch-ffi.checks.spec
+          nix-build -j 2 -A haskellPackages.hasktorch.checks.spec
+          nix-build -j 2 -A haskellPackages.examples.checks.spec
```

Comment:
Check success or failure of CI by reducing the number of jobs.

Reply:
looks like it's still failing